Fix the auth module registering a permanent cookie polling interval (#27851)
directus/directus: .changeset/cool-cookies-rest.md

---
'@directus/app': patch
---

> Fixed the auth module registering a cookie change listener at module scope, which left a permanent document.cookie

Reviewer: Nit
```suggestion
Fixed the auth module registering a permanent cookie polling interval
```

Author: Applied.
